com.librumreader.librum.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: com.librumreader.librum
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: librum
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=home
+  - --device=dri
+modules:
+  - python3-setuptools_scm.yaml
+  - python3-scikit-build.yaml
+  - python3-swig.yaml
+  - python3-libclang.yaml
+  - name: librum
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release 
+      - -DBUILD_TESTS=Off
+    sources:
+        - type: git
+          url: https://github.com/Librum-Reader/Librum.git
+          commit: 13386c7185f37e1f9525f86ba959d5fcf1e0f7ef
+  - name: install-files
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 librum.desktop -t /app/share/applications
```

Comment:
the .desktop file should not use an absolute path for the `Excec`. You need to change it with `desktop-file-edit` until it's fixed upstream.

Reply:
Fixed upstream